lms/resources/_js_config/__init__.py

```diff
@@ -63,14 +63,24 @@ def add_document_url(self, document_url):
                     _query={"document_url": document_url},
                 ),
             }
-        elif document_url.startswith("canvas://"):
+        elif document_url.startswith("canvas://file"):
```

Comment:
We could have collapsed these two in one for `canvas` and then specialize on `file` and `page`.

I reckon the two branches is easier to follow and modify in the future.